### PR TITLE
docs: align editorial typography docs with figma

### PR DESCRIPTION
**Pull request by frshwtr**
## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro [hunk 1]
```diff
@@ -21,121 +21,99 @@ const sbCaption = sbDomain + '--caption';
...
-			<td class="body-s">body</td>
-			<td>
-				<FigmaSbLinks figma={figmaBody} storybook={sbBody} />
-			</td>
-		</tr>
-		<tr>
-			<td>L</td>
-			<td class="body-l">body</td>
-			<td>
-				<FigmaSbLinks figma={figmaBody} storybook={sbBody} />
```
Comment (akomiqaia) [inline apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro:42]: Are links not useful here?

File: apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro [hunk 2]
```diff
@@ -21,121 +21,99 @@ const sbCaption = sbDomain + '--caption';
...
+	</tr>
 	</tbody>
 </table>
 
 <h4>Details</h4>
 <p>
-	The following styles have the same values across all screen sizes, mobile,
-	tablet, desktop.
+	The following styles have the same values across all screen sizes: mobile,
+	tablet, and esktop.
```
Comment (akomiqaia) [inline apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro:58]:
```suggestion
	tablet, and desktop.
```

**Review by akomiqaia — APPROVED** [review]
LGTM just one typo and a question.

File: apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro (hunk 1 above)
Comment (frshwtr) [reply apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro:42]: @akomiqaia This was following what was in [Figma](https://www.figma.com/design/y25ZjT1MqqHbNXNXsm8DET/Documentation-Content-Template?node-id=139-548&t=6hp2KFL1EchgbpZh-0)

**Review by notlee — CHANGES_REQUESTED** [review]
Thanks for picking up this ticket in my absence, but it's not what Mary and I had intended. It's a faithful implementation of what's in Figma, but it is intended only to convey content – it's not a website design.

- We aimed to replace tables with Previews to reduce repetition and allow engineers to copy HTML/JSX. For example we intended to present the three bylines elements as one in a Preview.
- This hard codes values (e.g. body is Georgia / regular / scale 2).  It's inaccurate, as it's not true for all brands. It is hard to maintain. We should use tokens to document values but this is also quite expensive to maintain. Me and Mary discussed costs and benefits, and agreed to remove the values here for now.
- Accessibility notes: This is superfluous and should be deleted. Someone would have to try hard for lists not to be navigable by keyboard.

Let me know if you'd like to discuss this. Since Mary and I previously went over the content changes, I'd be happy to pick this back up now I'm back if you like

File: apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro [hunk 3]
```diff
@@ -1,154 +1,46 @@
...
-		<tr>
-			<td>L</td>
-			<td class="body-l">body</td>
-			<td>
-				<FigmaSbLinks storybookId={storybookIdBody} />
-			</td>
-		</tr>
-	</tbody>
-</table>
+<p>Use for body text, paragraphs and lists. This is responsive.</p>
```
[the remark this answers is not in the document]
Comment (frshwtr) [reply apps/for-everyone-website/src/components/editorial-typography/BodyStyles.astro]: I cannot approve (as discussed on standup) as i opened this. My review:

* `BodyStyles.astro` still exists, can the previews still be moved to mdx?

**Pull request merged**